docs/ui/components/Home/index.tsx

```diff
@@ -52,12 +52,12 @@ export const APIGridCell = ({
   <Col css={cellWrapperStyle} md={md} sm={sm} xl={xl}>
     <div css={[cellStyle, cellAPIStyle, style]}>
       <div css={cellIconWrapperStyle}>{icon}</div>
-      <div css={cellTitleWrapperStyle}>
-        <Link href={link} css={cellTitleStyle}>
+      <Link href={link} css={cellTitleWrapperStyle}>
+        <>
```

Comment:
Do we need the fragment for `Link` components? I don't think we need that, because it's being wrapped in an anchor tag inside the link component 😄 Or are the types complaining?

Reply:
Nope, it's probably a leftover from refactor, because I have tried few approaches here, great catch! 😃